docs/DEVELOPMENT.md

```
@@ -42,89 +42,102 @@ bun run zip:firefox   # Firefox
-│   └── tocGenerator.ts         # Table of contents feature
-├── types/               # TypeScript Definitions
-│   ├── feature.ts       # Feature type definitions
-│   ├── hideable.ts      # Hideable element type definitions
-│   └── settings.ts      # Settings type definitions
-└── utils/               # Utility modules
-    ├── pageDetector.ts  # Page type detection
-    └── storage.ts       # Storage management
+├── assets/                  # Static assets (icons)
+├── config/                  # Configuration
+│   └── selectors.ts         # Centralized CSS selectors
+├── entrypoints/             # Extension Entry Points
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)